readyapi4j-core/src/main/java/com/smartbear/readyapi4j/assertions/ValidHttpStatusCodesAssertionBuilder.java

```diff
@@ -9,7 +9,7 @@
 
 import static com.smartbear.readyapi4j.Validator.validateNotEmpty;
 
-public class ValidHttpStatusCodesAssertionBuilder<T extends Assertion>  implements HttpStatusCodeAssertionBuilder {
+public class ValidHttpStatusCodesAssertionBuilder<T extends Assertion> implements HttpStatusCodeAssertionBuilder {
```

Comment:
Now that InvalidHttpStatusCodesAssertionBuilder doesn't extend this class. We should remove generics from here and update the code accordingly.